homeworks/kostiantyn.nosatskyi_NosatskyiK/PokemonProject/style.css

```diff
@@ -0,0 +1,28 @@
+h1 {
+  font-size: 48px ;
+  font-family: Roboto, serif ;
+  font-weight: 700 ;
+}
+
+.main_text {
+  font-family: Roboto, serif ;
+  font-size: 22px ;
+  font-weight: 400 ;
+}
+
+/* додав розміри для article щоб відповідати макету */
+
+article {
+  width: 575px ;
+  height: 479px ;
+}
+
+.link_MOC {
+  text-decoration: underline ;
+  color: #EF4934 ;
+}
+
+.link_social {
+  text-decoration: none ;
```

Comment:
How do I, as a user of your site understand, that this is the link? I beleive, that all links should look similar (to the ma-link)
<img width="770" alt="image" src="https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/b124a31a-86e2-40a4-94a0-c314b5842147">